plugins/output/flight/flight.go

```diff
@@ -131,11 +99,124 @@ func (f *Flight) Connect() error {
 	return nil
 }
 
+// Write should write immediately to the output, and not buffer writes
+// (Telegraf manages the buffer for you). Returning an error will fail this
+// batch of writes and the entire batch will be retried automatically.
+func (f *Flight) Write(metrics []telegraf.Metric) error {
+
+	// Create a new RecordBuilder using the schema.
+	builder := array.NewRecordBuilder(memory.DefaultAllocator, &f.schema)
+	defer builder.Release()
+
+	schemaFields := f.schema.Fields()
+
+	builder.Reserve(len(metrics))
+
+	// Iterate through the metrics and add them to the RecordBuilder.
+	for _, metric := range metrics {
+
```

Comment:
Very minor, but maybe remove this empty line to be consistent with the loop in `addMetricToRecordBuilder`.

Reply:
I think this is fixed now, was this what you meant?